### PR TITLE
Switch to using deep links

### PR DESCRIPTION
**Pull request by selvach**
Switch to using deep links on the pop up that forces users to upgrade to the latest version. 

Inspired by this Stack Overflow:
https://stackoverflow.com/questions/62445835/how-to-open-directly-our-app-in-apple-store

### CODE REVIEW

File: ios/Covid/Info.plist [hunk 1]
```diff
@@ -37,6 +37,10 @@
 	<string>4</string>
 	<key>ITSAppUsesNonExemptEncryption</key>
 	<false/>
+	<key>LSApplicationQueriesSchemes</key>
+    <array>
+        <string>itms-apps</string>
```
Comment (bionara) [inline ios/Covid/Info.plist:42]: should this e items-apps?

**Review by bionara — APPROVED** [review]
approved with pending typo(?)

File: ios/Covid/Info.plist (hunk 1 above)
Comment (selvach) [reply ios/Covid/Info.plist:42]: `itms-apps` is correct. From what I have googled - it's the Apple Store specific url schema for deep linking to the App Store:

https://developer.apple.com/forums/thread/104915

https://stackoverflow.com/questions/47348083/what-is-the-difference-between-itms-app-and-https-app-store-link

Comment (bionara) [reply ios/Covid/Info.plist:42]: ace

**Pull request merged**